test/Makefile

```diff
@@ -212,9 +212,6 @@ run-l2-explorer: ## Runs L2 blockscan explorer
 	$(RUNEXPLORERJSONRPC)
 	$(RUNEXPLORERL2)
 
-.PHONY: run-explorer
-run-explorer: run-explorer-l1 run-explorer-l2 ## Runs both L1 and L2 explorers
-
```

Comment:
Why did you remove this? please keep it

Reply:
because this command was not working. `run-explorer-l1` was removed in another PR. Also, to run `run-explorer-l2` we have run-l2-explorer